frontend/src/styles/resetDefaults.css

```diff
@@ -0,0 +1,89 @@
+/* "Meyer" CSS reset - useful for starting with a blank slate  */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
```

Comment:
I really like adding this rule to all my projects:

```css
*, *:before, *:after {
  box-sizing: border-box
}
```

The TL:DR; is that it makes sure padding is included in all width/height calculations. Consider this style:

```
.box {
  width: 100%;
  padding: 2rem;
}
```

Normally, this box would take up 100% + 16px of the parent container, which is never what you want. The box-sizing rule I shared ensures that it actually stays at 100%.